src/main/java/org/example/HomePage.java

```diff
@@ -0,0 +1,79 @@
+package org.example;
+
+import org.checkerframework.checker.units.qual.A;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.bidi.network.Header;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import javax.swing.*;
+
+public class HomePage extends BasePage {
+
+     @FindBy (css = "[id='userBadge']")
+     WebElement header;
+
+     @FindBy (xpath = "//section//i[@data-testid='sidebar-create-playlist-btn']")
+     WebElement PlaylistButton;
+
+     @FindBy (css = "[data-testid='playlist-context-menu-create-simple']")
+     WebElement CreateNewPlaylist;
+
+     @FindBy (css = "[name='name']")
+     WebElement NewPlayList;
+
+    @FindBy(xpath = "//ul/li[contains(text(),'Edit')]")
+     WebElement editButton;
+
+    @FindBy (xpath = "//ul/li//input[@type='text']")
+    WebElement renamePlayListInput;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```